tests/cli/commands/test_connection_command.py

```diff
@@ -46,15 +47,17 @@ def setup_method(self):
 
     def test_cli_connection_get(self):
         with redirect_stdout(io.StringIO()) as stdout:
-            connection_command.connections_get(
-                self.parser.parse_args(["connections", "get", "google_cloud_default", "--output", "json"])
-            )
+            # connection_command.connections_get(
+            #     self.parser.parse_args(["connections", "get", "google_cloud_default", "--output", "json"])
```

Comment:
@blag I modified the test in this way to support running the test with current code that supports click. Could you share your review on this part?